packages/financial-templates-lib/src/price-feed/CryptoWatchPriceFeed.js

```diff
@@ -109,12 +109,32 @@ class CryptoWatchPriceFeed extends PriceFeedInterface {
       return time < pricePeriod.closeTime && time >= pricePeriod.openTime;
     });
 
-    if (match === undefined) throw new Error(`Cryptowatch didn't return an ohlc for time ${time}`);
+    if (match === undefined) {
+      const before = this.historicalPricePeriods
+        .slice()
```

Comment:
why do you have to `slice()` this?

Reply:
`slice` is an easy way to grab a copy of an array. `reverse` modifies the array in-place, so I copied before calling it as to not corrupt the data.

Reply:
ah cool i was wondering if this modified the array in place.